composer.json

```diff
@@ -37,17 +37,30 @@
             "homepage": "http://superdweebie.com"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/webimpress/DoctrineModule"
+        }
+    ],
     "require": {
-        "php": "~5.5 || ~7.0",
-        "doctrine/doctrine-module": "~1.0",
-        "doctrine/mongodb-odm": "~1.0",
-        "zendframework/zend-mvc": "2.*",
-        "zendframework/zend-servicemanager": "2.*",
-        "zendframework/zend-stdlib": "2.*"
+        "php":                               "~5.6 || ~7.0",
```

Comment:
hm .... https://github.com/doctrine/DoctrineModule/blob/master/composer.json#L33-L56
I'd like just to be consistent with your other libraries ;) What do you think then?